pages/about/getting-in-touch.md

```diff
@@ -2,12 +2,25 @@
 layout: page
 title: Getting in touch
 permalink: /getting-in-touch
-nav_order: 2
+nav_order: 10
```

Comment:
Page getting in touch now at the very bottom? You feel that's not better to have it higher up? No strong feeling.

Reply:
I'll have to check the actual order, I'll post a screenshot in a bit - I'm spreading the numbers a bit so it's easier to inject a page without renumbering everything.

Reply:
OK, fine.